doc/quickstart.md

```diff
@@ -0,0 +1,111 @@
+# vfkit quick start
+
+## Introduction
+
+vfkit is a macOS command line hypervisor.
```

Comment:
`command line-based` ?

Reply:
`vfkit is a command line-based hypervisor for macOS.`

Reply:
I changed to `commandline-based` hypervisor, I think I use "commandline" in a single word in other places. Looks like it should be `command-line` though https://en.wikipedia.org/wiki/Command-line_interface , I'll update this!